test/functional/tests/cache_ops/test_cleaning_policy_operation.py

```diff
@@ -0,0 +1,261 @@
+#
+# Copyright(c) 2019 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+
+
+import time
+import pytest
+
+from datetime import timedelta
+from api.cas import casadm
+from api.cas.cache_config import (
+    CacheMode,
+    CleaningPolicy,
+    FlushParametersAcp,
+    FlushParametersAlru,
+    Time,
+)
+from storage_devices.disk import DiskType, DiskTypeSet, DiskTypeLowerThan
+from core.test_run import TestRun
+from test_utils.size import Size, Unit
+from test_utils.os_utils import Udev, sync
+from test_tools.fio.fio import Fio
+from test_tools.fio.fio_param import ReadWrite, IoEngine
+
+
+cores_count = 4
+io_size = Size(10000, Unit.Blocks4096)
+# time_to_wait in seconds
+# For 4 cores and io_size = 10000 Blocks4096, 30 seconds of waiting should be enough
+# for CAS cleaner to flush enough data for test purposes.
+time_to_wait = 30
+
+
+@pytest.mark.parametrize("cleaning_policy", CleaningPolicy)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_cleaning_policies_in_write_back(cleaning_policy):
+    """
+        title: Test for cleaning policy operation in Write-Back cache mode.
+        description: |
+          Check if ALRU, NOP and ACP cleaning policies preserve their
+          parameters when changed and if they flush dirty data properly
+          in Write-Back cache mode.
+        pass_criteria:
+          - Flush parameters are set to different values if applied.
```

Comment:
Not very clear criterium - I had to guess what you had in mind

Reply:
Done